### ISSUE TITLE
Create current Android mockup on Proto.io

### ISSUE THREAD

**Issue by jekram** [Cloudkibo/Intern#27]
@AsadUllahBinTahir 

I would like you to create the current mock up of Android implementation on Proto.io

Please see @sojharo for current Android implementation and he will give you all the screenshots.

I would like you to import that into Proto.io so we can make improvement. Please create these on this account:
  https://docs.google.com/spreadsheets/d/1hctAh6run3Q7eHZPKiD5sTe6gCDZefbHNkR1O1NVGDw/edit#gid=0

**Comment by AsadUllahBinTahir** [comment]
Acknowledged. I will try complete it tomorrow.

**Comment by jekram** [comment]
Thanks

**Comment by sojharo** [comment]
There is some migration problem in Android application. When I used to develop Android on Windows, there were some keys stored on Windows C:\ directory for Facebook Login. Now, I am on Android so I have to recreate these keys and put it in Linux directory. I would look into this issue day after tomorrow.

I have just seen this issue as Asad told me about this.

**Comment by jekram** [comment]
@sojharo 

WE should track this issue on Andriod repo.

**Comment by sojharo** [comment]
Yes, I would open the issue on Android repo for this. Also there is one test by @ImranBinShoukat which will get block because of this. It is on #26 .

I need Sunday to look into this issue else it would waste time for other tasks. On Sunday, I would fix this and both Imran and Asad would be able to continue there tasks related to Android.

**Comment by jekram** [comment]
Thanks. That is fine.

**Comment by jekram** [comment]
@sojharo 

Can you unblock this issue so @AsadUllahBinTahir can complete his assignment

**Comment by jekram** [comment]
@sojharo 

Can they work on this issue now? We need to have them work on it so we can change the UI.

What is the next step

**Comment by sojharo** [comment]
Yes, I would add them as testers in the store. So that they can install it on their own without waiting on me. I would add new versions to the store directly.

**Comment by sojharo** [comment]
Here is the link to download android app:

https://play.google.com/apps/testing/com.cloudkibo

**Comment by jekram** [comment]
@sojharo @AsadBinTahir 

Thanks

**Comment by sojharo** [comment]
@AsadBinTahir @ImranBinShoukat 

Please give me your gmail ids or cloudkibo ids.

Google Play Store doesn't allow me to add your current IBA email addresses as testers.

**Comment by jekram** [comment]
Their cloudkibo id's are

asad@cloudkibo.com
imran@cloudkibo.com

@AsadBinTahir @AsadBinTahir 

Please also send your emails

**Comment by sojharo** [comment]
Both the cloudkibo ids are added as testers to app store.
